services/web/server/src/simcore_service_webserver/activity/celery_client.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import tenacity
+from aiohttp import web
+from celery import Celery
+from tenacity import before_log, stop_after_attempt, wait_fixed
+
+from ..computation_config import ComputationSettings
+from ..computation_config import get_settings as get_computation_settings
+from .config import APP_CLIENT_CELERY_CLIENT_KEY
```

Comment:
MINOR: one thing I started doing is hiding this constant in the module  (i.e. ``__APP_CLIENT_CELERY_CLIENT_KEY__``  so it enforces the developer to use ``get_celery_client``.

Reply:
done